script/cargo.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+
```

Comment:
Please run `npm run pep8` inside src/brave and then fix the python lint problems.

Reply:
👍 lint problems fixed and my tabs updated to 4 spaces